src/spikeinterface/postprocessing/amplitude_scalings.py

```diff
@@ -291,10 +365,325 @@ def _amplitude_scalings_chunk(segment_index, start_frame, end_frame, worker_ctx)
             else:
                 local_waveform = traces_with_margin[cut_out_start:cut_out_end, sparse_indices]
             assert template.shape == local_waveform.shape
-            local_waveforms.append(local_waveform)
-            templates.append(template)
+
             linregress_res = linregress(template.flatten(), local_waveform.flatten())
-            scalings.append(linregress_res[0])
```

Comment:
why having scipy.linregress and sklearn.linear_model.LnearRegression for colliding spikes ?
Does scipy can make the same ?

Reply:
yes, but we can unify that

Reply:
So scipy is the fastest and it gives the exact same result (to the 6th decimal) of `LinearRegression`. 

![image](https://github.com/SpikeInterface/spikeinterface/assets/17097257/74fd4f5a-03bc-4f56-8e9b-c18f4d230140)

The `_prms` version here is with `fit_intercept=False, positive=True`. Using `positive=True` is the same as without, but slightly faster (255 ms ± 1.64 ms VS 286 ms ± 3 ms).
In my opinion it's correct to also fit the intercept for a better fit, even to be more robust against some fluctuiations.

What should we do?